src/payloads/Attendance.ts

```diff
@@ -1,24 +1,22 @@
-import { ValidateNested, IsOptional, IsBoolean, IsNumber, IsInstance } from 'class-validator';
-import { AppUser, Event } from '@Entities';
+import { IsOptional, IsBoolean, IsNumber, IsInstance } from 'class-validator';
+import { AppUserEventResponse } from './AppUser';
+import { EventAttendanceResponse } from './Event';
 
 abstract class BaseAttendancePayload {
-  @ValidateNested({ each: true })
-  @IsInstance(AppUser)
-  readonly attendee: AppUser;
+  @IsInstance(AppUserEventResponse)
```

Comment:
also feels a little weird to have response in the base payload esp since you'll possibly have attendance request later which would also extend BaseAttendancePayload